crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -47,11 +47,39 @@ pub struct Mesh {
 /// ```
 /// # use bevy_render::mesh::{Mesh, Indices};
 /// # use bevy_render::render_resource::PrimitiveTopology;
-/// fn create_triangle() -> Mesh {
+/// fn create_simple_parallelogram() -> Mesh {
+///     // create a new mesh, add 4 vertices, each with its own position attribute (coordinate in
+///     // 3D space). for each of the points of the parallelogram. for more examples of different
+///     // shapes, see the built in examples in bevy.
```

Comment:
Make sure that every sentence start with an uppercase character, it seems to be the convention in Bevy's code/doc.

"for more examples of different shapes, see the built in examples in bevy" is redundant with the explanation bellow the example, should be removed.